### PR TITLE
Support new poloniex api

### PR DESCRIPTION
**Pull request by LefterisJP**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/tests/api/test_history.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 @pytest.mark.parametrize(
     'added_exchanges',
     [(Location.BINANCE, Location.POLONIEX, Location.BITTREX, Location.BITMEX, Location.KRAKEN)],
+    # [(Location.BINANCE, Location.BITTREX, Location.BITMEX, Location.KRAKEN)],
```
Comment (yabirgb) [inline rotkehlchen/tests/api/test_history.py:32]: If is not needed can be removed

File: rotkehlchen/types.py [hunk 2]
```diff
@@ -68,7 +68,6 @@
 YEARN_VAULTS_V2_PROTOCOL = 'yearn_vaults_v2'
 CURVE_POOL_PROTOCOL = 'curve_pool'
 PICKLE_JAR_PROTOCOL = 'pickle_jar'
-COMPOUND_PROTOCOL = 'compound'
```
Comment (yabirgb) [inline rotkehlchen/types.py:71]: why was this removed?

**Review by yabirgb — COMMENTED** [review]
Some questions

File: rotkehlchen/types.py (hunk 2 above)
Comment (LefterisJP) [reply rotkehlchen/types.py:71]: Because it's not used anywhere?

File: rotkehlchen/tests/api/test_history.py (hunk 1 above)
Comment (LefterisJP) [reply rotkehlchen/tests/api/test_history.py:32]: Ah fuck this was a leftover. Will remove it later
